src/main/java/org/broadinstitute/hellbender/engine/filters/MetricsReadFilter.java

```diff
@@ -4,6 +4,15 @@
 import org.broadinstitute.hellbender.utils.help.HelpConstants;
 import org.broadinstitute.hellbender.utils.read.GATKRead;
 
+/**
+ * Filter out reads that:
+ *
+ * <ul>
+ *     <li>Fail platform/vendor quality checks</li>
```

Comment:
True, but I think that they add some valuable information for the user. I added them, but I could remove them in the next pass...